projects/mercury/src/file/DirectoryInformationDrawer.js

```diff
@@ -254,8 +254,11 @@ const PathMetadata = React.forwardRef((
 ) => {
     const {data, error, loading} = useAsync(() => LocalFileAPI.stat(path, showDeleted), [path]);
     const {hierarchy, vocabulary} = useContext(VocabularyContext);
-    const [updateDate, setUpdateDate] = useState(Date.now());
+    const [updateDate, setUpdateDate] = useState(undefined);
```

Comment:
You can just use `useState()`, the argument is not required, if it's undefined by default